qutip/tests/test_bloch.py

```diff
@@ -419,3 +419,8 @@ def test_vector(self, vector_kws, fig_test, fig_ref):
             vector_kws = [vector_kws]
         self.plot_vector_test(fig_test, copy.deepcopy(vector_kws))
         self.plot_vector_ref(fig_ref, copy.deepcopy(vector_kws))
+
+
+def test_repr_svg():
+    svg = Bloch()._repr_svg_()
+    assert isinstance(svg, str)
```

Comment:
We could probably make this test a little stronger -- e.g. `assert svg.startswith(...)` and maybe `.endswith(...)`.

Reply:
Great idea! I added it into the test.